File: creacionTurnos/src/interfaces/turnos.ts
export interface Turnos {

  idTurno:string;
  nombreComercio:string;
  nombreServicio:string;
  fechaTurno:string;
  horaInicio:string;
  horaFin:string;
  estado:string;
}

export interface Comercio {

  nombreComercio:string;
  aforoMaximo:number;
}

export interface Servicios {

  nombreComercio:string;
  nombreServicio:string;
  horaApertura:number;
  horaCierre:number;
  duracion:string;

}

export class RequestLeerServicio{
  comercio:string;
  constructor(comercio:string){
      this.comercio = comercio;
  }
}


export class RequestGuardarTurno{
  nombreComercio:string;
  nombreServicio:string;
  fechaInicio:string;
  fechaFin:string;
  constructor(nombreComercio:string,nombreServicio:string,fechaInicio:string,fechaFin:string){
      this.nombreComercio=nombreComercio;
      this.nombreServicio=nombreServicio;
      this.fechaInicio=fechaInicio;
      this.fechaFin=fechaFin;
  }
}

File: turnos/src/main/java/com/reserva/turnos/dto/Turnos.java
package com.reserva.turnos.dto;

import lombok.Builder;
import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.Date;

@Data
@Builder
@Document("Turnos")
public class Turnos {

    @Id
    private String idTurno;
    private String nombreComercio;
    private String nombreServicio;
    private String fechaTurno;
    private String horaInicio;
    private String horaFin;
    private String estado;

}
